app/models/concerns/single_receptacle_labware.rb

```diff
@@ -22,8 +22,6 @@ module SingleReceptacleLabware
     delegate :aliquots, :aliquots=, to: :receptacle
     delegate :concentration, :concentration=, to: :receptacle
     # Delegate for the moment, update once things are working
-    delegate :transfer_requests_as_target, :transfer_requests_as_source, to: :receptacle
-    # Delegate for the moment, update once things are working
```

Comment:
don't need this because it's on Labware model?

Reply:
Yep, was only ever supposed to stay there while I refactored, a mistake to have left it in.